pkg/cmd/grafana-cli/utils/grafana_path.go

```diff
@@ -2,36 +2,41 @@ package utils
 
 import (
 	"os"
+	"path"
+	"path/filepath"
 
 	"github.com/grafana/grafana/pkg/cmd/grafana-cli/logger"
 )
 
 func GetGrafanaPluginDir(currentOS string) string {
-	//currentOS := runtime.GOOS
-
-	if currentOS == "windows" {
-		return returnOsDefault(currentOS)
-	}
-
-	pwd, err := os.Getwd()
-
-	if err != nil {
-		logger.Error("Could not get current path. using default")
-		return returnOsDefault(currentOS)
-	}
-
-	if isDevenvironment(pwd) {
+	if isDevEnvironment() {
 		return "../data/plugins"
 	}
 
 	return returnOsDefault(currentOS)
 }
 
-func isDevenvironment(pwd string) bool {
+func isDevEnvironment() bool {
 	// if ../conf/defaults.ini exists, grafana is not installed as package
 	// that its in development environment.
-	_, err := os.Stat("../conf/defaults.ini")
-	return err == nil
+	ex, err := os.Executable()
+	if err != nil {
+		logger.Error("Could not get executable path. Assuming non dev environment.")
+		return false
+	}
+	exPath := filepath.Dir(ex)
+	_, dir := path.Split(exPath)
```

Comment:
```suggestion
	dir, _ := path.Split(exPath)
```
I think the vars should be the other way around. 

https://github.com/golang/go/blob/go1.12.4/src/path/path.go#L147:6

Reply:
hmm the naming could be confusing here but we actually get a `filepath.Dir()` before that so the split returns `rest, tail` in that case or something like that. In this case `dir` will be `bin` and `_` will be rest of the path.

Reply:
Ah. got confused by the named return. `rest`, `tail` is more understandable. Perhaps you should send a PR to the go project? :)